### ISSUE TITLE
Missing aria-label for resource cells in tables

### ISSUE THREAD

**Issue by amisskii** [containers/podman-desktop#13291]
### Is your enhancement related to a problem? Please describe

Currently, the resource cells in our tables do not have aria-label attributes

### Describe the solution you'd like

Each resource cell would have  a meaningful aria-label
